fre/app/generate_time_averages/generate_time_averages.py

```diff
@@ -5,7 +5,22 @@
 def generate_time_average(infile = None, outfile = None,
                           pkg = None, var = None, unwgt = False,
                           avg_type = None):
-    ''' steering function to various averaging functions above'''
+    """
+    steering function to various averaging functions above
+    
+    :param infile: path to history file, or list of paths
+    :type infile: str, list
+    :param outfile: path to where output file should be stored
+    :type outfile: str
+    :param pkg: which package to use to calculate climatology (cdo, fre-nctools, fre-python-tools)
+    :type pkg: str
+    :param var: not currently supported
+    :type var: defaults to none
+    :param unwgt: wether or not to weight the data
+    :type unwgt: True or False, default false
```

Comment:
I think we would say the type is `bool`, move the default to the param description (line above)